src/mbed_tools/project/_internal/templates/main.tmpl

```diff
@@ -0,0 +1,13 @@
+/* mbed Microcontroller Library
+ * Copyright (c) {{year}} ARM Limited
```

Comment:
Will it support the year format `2020 - 2021` or `2020,2021` etc

Reply:
Don't think it needs to. This is just the year of the creation of the file.